### PR TITLE
add colander

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://pypi.org/project/colander/

### CODE REVIEW

**Comment by ocefpaf** [comment]
@conda-forge/staged-recipes this one is ready for review!

File: recipes/colander/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - colander
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Pylons/colander
+  summary: A simple schema-based serialization and deserialization library
+  license: Mup
```
Comment (BastianZim) [inline recipes/colander/meta.yaml:39]: It's similar but I don't think it's actually `Mup`: https://spdx.org/licenses/Mup.html

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/colander/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/colander/meta.yaml:39]: They say BSD-derived: https://github.com/Pylons/colander/blob/master/setup.cfg#L8

Looks like it is this one: https://fedoraproject.org/wiki/Licensing:BSD#Modification_Variant

But I could not find a SPDX form for it.

File: recipes/colander/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Pylons/colander
+  summary: A simple schema-based serialization and deserialization library
+  license: LicenseRef-BSD-ModificationVariant
+  license_file: COPYRIGHT.txt, LICENSE.txt
```
Comment (BastianZim) [inline recipes/colander/meta.yaml:39]:
```suggestion
  license_file: 
    - COPYRIGHT.txt
    - LICENSE.txt
```
The original  is valid yaml but converts it to a string and not a list.

File: recipes/colander/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/colander/meta.yaml]: Yep. Looks like it. Thanks!

**Comment by BastianZim** [comment]
Hmm that linter complaint is strange – it's a valid expression. If you're happy with the recipe I can merge it and then we can file an issue in the linter repo.

File: recipes/colander/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Pylons/colander
+  summary: A simple schema-based serialization and deserialization library
+  license: Mup
+  license_file: COPYRIGHT.txt
```
[the remark this answers is not in the document]
Comment (marcelotrevisani) [reply recipes/colander/meta.yaml]: indeed, I think it is because that file was the first that grayskull found, but indeed we can improve that

**Comment by ocefpaf** [comment]
> Hmm that linter complaint is strange – it's a valid expression. If you're happy with the recipe I can merge it and then we can file an issue in the linter repo.

We need to add that license to the list of valid ones. We can do that later. Merge when you are ready. Thanks!

**Comment by BastianZim** [comment]
Yeah I asked in gitter - seems like an extra license package would be better but that might be a first step.

**Pull request merged**